wolfcrypt/src/misc.c

```diff
@@ -254,6 +254,32 @@ WC_MISC_STATIC WC_INLINE void writeUnalignedWords32(byte *out, const word32 *in,
 
 #if defined(WORD64_AVAILABLE) && !defined(WOLFSSL_NO_WORD64_OPS)
 
+#ifdef WOLFSSL_X86_64_BUILD
+
+WC_MISC_STATIC WC_INLINE word64 readUnalignedWord64(const byte *in)
```

Comment:
Interesting... I wonder if there are any GCC builtin_ functions for this?

Reply:
douzzer decided to add these.
There are memcpy unaligned builtins.